### PR TITLE
#20 - categorize EPISB documents

### PR DESCRIPTION
**Pull request by oddodaoddo**
@nsheff : take a look and see if this suits.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -8,17 +8,19 @@ edit_uri: edit/dev/docs/
 nav:
   - Getting started:
     - Introducing episb: README.md
-  - How-to guides:
+  - HUB user guides:
```
Comment (nsheff) [inline mkdocs.yml]: no need to capitalize HUB

File: mkdocs.yml [hunk 2]
```diff
@@ -8,17 +8,19 @@ edit_uri: edit/dev/docs/
...
     - Query types explained: howto-query-provider.md
     - Add a new provider to a hub: howto-subscribe.md
     - Normalize to a segmentation: howto-normalize.md
+  - HUB/Provider administrator guides:
     - Build a data provider: howto-build-provider.md
     - Build a hub: howto-build-hub.md
+  - HUB/Provider programmer's guides:
+    - Hub API: hub-api.md
```
Comment (nsheff) [inline mkdocs.yml]: API documentation should clearly go under reference, I think...

A howto would be a bit more...friendly.

ideally the API documentation can be auto-generated from code comments.

File: mkdocs.yml [hunk 3]
```diff
@@ -8,17 +8,19 @@ edit_uri: edit/dev/docs/
...
     - Access data using R or python: howto-use-episb-API-calls.md
+    - Data provider API: provider-api.md
```
Comment (nsheff) [inline mkdocs.yml]: again I think the API should be Reference... 

a Howto is a tutorial. it walks the user through "how to" do something. there is a target. API documentation does not fit that.

File: mkdocs.yml [hunk 4]
```diff
@@ -8,17 +8,19 @@ edit_uri: edit/dev/docs/
 nav:
   - Getting started:
     - Introducing episb: README.md
-  - How-to guides:
+  - HUB user guides:
     - Query types explained: howto-query-provider.md
     - Add a new provider to a hub: howto-subscribe.md
     - Normalize to a segmentation: howto-normalize.md
+  - HUB/Provider administrator guides:
```
Comment (nsheff) [inline mkdocs.yml]: how about "User guides" and "Administrator guides".

File: mkdocs.yml (hunk 1 above)
Comment (oddodaoddo) [reply mkdocs.yml]: I think I will just do away with the word "hub" as it is. See new commit.

File: mkdocs.yml (hunk 2 above)
Comment (oddodaoddo) [reply mkdocs.yml]: Got it. Yes, we will auto-generate API docs - scalatra has that capability, I just need to turn it on and make it happen.

File: mkdocs.yml [hunk 5]
```diff
@@ -8,17 +8,19 @@ edit_uri: edit/dev/docs/
...
     - Introducing episb: README.md
-  - How-to guides:
+  - User guides:
     - Query types explained: howto-query-provider.md
     - Add a new provider to a hub: howto-subscribe.md
     - Normalize to a segmentation: howto-normalize.md
+  - Administrator guides:
     - Build a data provider: howto-build-provider.md
     - Build a hub: howto-build-hub.md
+  - Programmer's guides:
```
Comment (nsheff) [inline mkdocs.yml]: I like where you're going with this, but given there's only 1 programmer guide, let's just leave it under User guides... programmers are users, too.

**Comment by nsheff** [comment]
otherwise this looks great.

File: mkdocs.yml (hunk 5 above)
Comment (oddodaoddo) [reply mkdocs.yml]: Done :smile:

**Pull request merged**